ERCS/erc-725.md

```diff
@@ -309,10 +288,25 @@ interface IERC725Y /* is ERC165, ERC173 */ {
     function setData(bytes32 dataKey, bytes memory dataValue) external;
     function setDataBatch(bytes32[] memory dataKeys, bytes[] memory dataValues) external;
 }
+
+interface IERC725X  /* is ERC-165, ERC-173 */ {
+    event ContractCreated(uint256 indexed operationType, address indexed contractAddress, uint256 value, bytes32 indexed salt);
+    event Executed(uint256 indexed operationType, address indexed target, uint256 value, bytes4 indexed selector);
+
+    function execute(uint256 operationType, address target, uint256 value, bytes memory data) external payable returns(bytes memory);
+
+    function executeBatch(uint256[] memory operationsType, address[] memory targets, uint256[] memory values, bytes memory datas) external payable returns(bytes[] memory);
+}
+
 interface IERC725 /* is IERC725X, IERC725Y */ {
+
 }
+
 ```
 
+
 ## Copyright
 
-Copyright and related rights waived via [CC0](../LICENSE.md).
+Copyright and related rights waived via CC0.
```

Comment:
Please use the exact waiver from the [template](https://github.com/ethereum/EIPs/blob/master/eip-template.md?plain=1).